sdmetrics/demos.py

```diff
@@ -22,7 +22,7 @@ def load_demo(modality='multi_table'):
     """
     demo_path = pathlib.Path(__file__).parent / 'demos' / f'{modality}.pkl'
     with open(demo_path, 'rb') as demo_file:
-        return pickle.load(demo_file)
+        return pickle.load(demo_file)  # noqa: DUO103
```

Comment:
This is a very common use case within our code, loading the path that the user gave us, so I would rather move the ignore to the `setup.cfg` and make it standard